app/models/rbac.rb

```diff
@@ -104,10 +104,8 @@ def self.rbac_class(scope)
     nil
   end
 
-  def self.rbac_instance(obj)
-    return obj                if apply_rbac_to_class?(obj.class)
-    return obj.resource       if obj.kind_of?(MetricRollup) || obj.kind_of?(Metric)
-    nil
+  def self.pluck_ids(targets)
+    targets.pluck(:id) if targets
```

Comment:
Are the callers ready for a return value of `nil` when targets are nil?  Or should this return empty array instead?